packages/connect/CHANGELOG.md

```diff
@@ -1,15 +1,54 @@
-|             Package              | Stable | Canary |
-| :------------------------------: | :----: | :----: |
-|       npm @trezor/connect        | 9.2.4  |   -    |
-|     npm @trezor/connect-web      | 9.2.4  |   -    |
-| npm @trezor/connect-webextension | 9.2.4  |   -    |
+|             Package              | Stable |    Canary    |
+| :------------------------------: | :----: | :----------: |
+|       npm @trezor/connect        | 9.2.4  | 9.2.5-beta.1 |
+|     npm @trezor/connect-web      | 9.2.4  | 9.2.5-beta.1 |
+| npm @trezor/connect-webextension | 9.2.4  | 9.2.5-beta.1 |
 
-|     Deployment      | Stable | Canary |
-| :-----------------: | :----: | :----: |
-| connect.trezor.io/9 | 9.2.4  |   -    |
+|     Deployment      | Stable |    Canary    |
+| :-----------------: | :----: | :----------: |
+| connect.trezor.io/9 | 9.2.4  | 9.2.5-beta.1 |
 
 User the persistent link connect.trezor.io/9 to access the latest stable version of Connect Explorer.
 
+# 9.2.5-beta.1
+
+-   feat(connect): Add tag 258 support (90bf3a7)
+-   feat(connect): Conway certificates (ab003ce)
+-   feat(connect): applySettings - accept all validated params (65809d8)
+-   fix(connect): immediate switch from custom to default backend (b8348ca)
+-   fix(connect): remove getPublicKey coinInfo fallback (fb446f2)
+-   chore(connect): add new TS3 CA pubkeys and update timestamp (35486ba)
+-   chore(connect): update fw version number for cardano (d737d3c)
```

Comment:
maybe we could list exact fw version which is now requried for cardano calls. please @martykan 🙏

Reply:
I think that this information would be helpful for QA, they will be able to test it more specifically.